pymc_bart/bart.py

```diff
@@ -204,6 +204,16 @@ def preprocess_xy(
     if isinstance(X, (Series, DataFrame)):
         X = X.to_numpy()
 
+    try:
+        import polars as pl
+
+        if isinstance(X, (pl.Series, pl.DataFrame)):
```

Comment:
@aloctavodia you should not have to do any type checking if both dataframes have `to_numpy()` methods.

Reply:
On my phone, later I can try to make this clenaer. But the logic is that we also support other objects like numpy, or list. So this is not a check of polars vs pandas.